en/docs/consuming-services/consuming-a-sevice.md

```diff
@@ -47,106 +44,101 @@ The overview page of an API displays the subscribed versions of the API along wi
 
 ## Consume the API via your web application
 
-You can invoke the API/service using the credentials you created above. Since Choreo services are secured, you need to invoke the services securely with the help of your IdP. To invoke the API/service you created securely, you need to follow the steps below:
+To securely invoke the API/service, you need to use your Identity Provider (IdP). Follow these steps:
 
 1. Create a web application in Choreo.
-2. Create an OAuth application in the IdP. 
-3. Configure the web application to authenticate the API/service invocations from the IdP, and securely invoke the service/API in Choreo.
-4. Deploy the web application. 
-
-In this guide, you will be using the following products and examples:
+2. Create an OAuth application in the IdP.
+3. Configure the web application to authenticate API/service invocations using the IdP.
+4. Deploy the web application.
 
--  WSO2 Asgardeo as the IdP
-- [choreo-samples/reading-list-app/reading-list-front-end](https://github.com/wso2/choreo-samples/tree/main/reading-list-app/reading-list-front-end) as the web application. This is a React SPA that invokes the service using Axios. This web application is configured to read the
-[choreo-samples/reading-list-app/reading-list-service](https://github.com/wso2/choreo-samples/tree/main/reading-list-app/reading-list-service) as the service. You can modify this web application to read your service, or [deploy the choreo-samples/reading-list-app/reading-list-service](../quick-start-guides/deploy-a-web-application-that-consumes-a-backend-service.md#step-12-configure-the-sample-service-and-publish-it-as-a-rest-api) in Choreo and use it. 
+For this guide, we’ll use:
```

Comment:
Based on https://developers.google.com/style/person#address-the-reader-as-you, shall we use the second person instead of the first person.